main.c

```diff
@@ -873,6 +905,16 @@ int main(int argc, char *argv[]) {
 	// second roundtrip for xdg-output
 	wl_display_roundtrip(state.display);
 
+	if (output_boxes) {
+		struct slurp_output *box_output;
+		wl_list_for_each(box_output, &state.outputs, link) {
+			add_choice_box(&state, &box_output->logical_geometry);
+			struct slurp_box *new_box;
+			new_box = wl_container_of(state.boxes.prev, new_box, link);
+			new_box->label = strdup(new_box->label); // to avoid double free
```

Comment:
I'm not sure if it's better to do this, or intentionally leak the labels. I think leaking the labels would probably be ok, since at that point we've reached the end of the program anyway.

Reply:
I think `add_choice_box` should perform the `strdup` instead.